# HG changeset patch
# User Hartmut Gimpel <hg_code@gmx.de>
# Date 1445792321 0
#      Sun Oct 25 16:58:41 2015 +0000
# Node ID b4ceb06009e0a21e3130ea024e7e42671564ee7d
# Parent  c169479913543e55f47b7dc984e0c5f762f38b3f
Fix regression on color conversion by 84ca63c8a038 (patch #8709)

* scripts/image/private/colorspace_conversion_input_check.m: fix conversion
  from integer type images to class double.

diff --git a/scripts/image/private/colorspace_conversion_input_check.m b/scripts/image/private/colorspace_conversion_input_check.m
--- a/scripts/image/private/colorspace_conversion_input_check.m
+++ b/scripts/image/private/colorspace_conversion_input_check.m
@@ -57,15 +57,15 @@ function [in_arg, cls, sz, is_im, is_nd,
     in_arg = reshape (in_arg, [numel(in_arg)/3 3]);
   else
     is_im = false;
     is_nd = false;
   endif
 
   ## Convert to floating point (remember to leave class single alone)
   if (isinteger (in_arg))
-    in_arg = double (in_arg) / double (intmin (cls));
+    in_arg = double (in_arg) / double (intmax (cls));
     is_int = true;
   else
     is_int = false;
   endif
 
 endfunction
